students/models.py

```diff
@@ -39,24 +36,21 @@ class Student(Person):
                                on_delete=models.SET_NULL)
     invited = models.IntegerField(default=0, null=True)
 
-    @classmethod
-    def generate_instances(cls, count):
-        faker = Faker()
-        for _ in range(count):
-            st = cls(
-                first_name=faker.first_name(),
-                last_name=faker.last_name(),
-                email=faker.email(),
-                birthdate=faker.date_time_between(start_date="-30y", end_date="-18y"),
-            )
-            st.save()
+    @receiver(post_save, sender=User)
+    def create_user_profile(sender, instance, created, **kwargs):
+        if created:
+            Profile.objects.create(user=instance)
+
+    @receiver(post_save, sender=User)
+    def save_user_profile(sender, instance, **kwargs):
+        instance.profile.save()
 
     def age(self):
         return datetime.datetime.now().year - self.birthdate.year
```

Comment:
Please correct it in models, because now we have current date as a data from migrations.